### PR TITLE
upgrade/modifications to treat Elastic data

### PR DESCRIPTION
**Pull request by marcol480**
**IMPORTANT: Please create an issue first before opening a Pull Request.**
Linked to issue(s): Issue #183

## What changes were proposed in this pull request?

Proposing to add relevant lines for treating Elastic data 
(part of the update is in a separate repository : Emille's repo )

## How was this patch tested?
Some tests failed

### CODE REVIEW

File: fink_science/random_forest_snia/processor.py [hunk 1]
```diff
@@ -352,15 +353,15 @@ def rfscore_sigmoid_elasticc(midPointTai, filterName, psFlux, psFluxErr, cdsxmat
...
+        model = curdir + '/data/models/model_elastic_marco20220708.obj'
         clf = load_scikit_model(model)
 
     test_features = []
     flag = []
     for id in np.unique(pdf['SNID']):
         f1 = pdf['SNID'] == id
         pdf_sub = pdf[f1]
-        features = get_sigmoid_features_dev(pdf_sub)
+        features = get_sigmoid_features_elastic(pdf_sub)
```
Comment (JulienPeloton) [inline fink_science/random_forest_snia/processor.py:364]: You forgot to import the function ;-)

**Review by JulienPeloton — COMMENTED** [review]
Thanks @marcol480 -- I left one minor comment below, and then I'll merge the code

File: fink_science/random_forest_snia/processor.py (hunk 1 above)
Comment (marcol480) [reply fink_science/random_forest_snia/processor.py:366]: Sorry, I just added this

**Comment by JulienPeloton** [comment]
Thanks @marcol480 -- I am merging despite the test failures. They will be fixed in another PR (that requires modifications in https://github.com/emilleishida/fink_sn_activelearning/pull/6)

**Pull request merged**